app/modules/intelligence/agents_copy/custom_agents/custom_agent_router.py

```
@@ -0,0 +1,119 @@
+)
+from app.modules.utils.APIRouter import APIRouter
+from app.modules.utils.logger import setup_logger
+
+logger = setup_logger(__name__)
+router = APIRouter()
+
+
+@router.post("/", response_model=Agent)
+async def create_custom_agent(
+    request: AgentCreate,
+    db: Session = Depends(get_db),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)